src/entities/PoolNetwork.ts

```diff
@@ -105,7 +105,7 @@ export class PoolNetwork extends Entity {
               client: this._root.getClient(this.chainId)!,
             })
             const results = await Promise.allSettled(
-              currencies.map(async (curAddr) => {
+              currencies.map(async (curAddr: any) => {
```

Comment:
currencies are not in the protocol addresses, so we need to hardcode them somewhere